app/build.gradle

```diff
@@ -72,6 +72,8 @@ dependencies {
 
     implementation "androidx.constraintlayout:constraintlayout:$versions.constraint_layout"
 
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
We don't require this dependency, please remove it.

Reply:
added androidx lib for swiperefresh